recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "0.17.2" %}
+{% set name = "myst-nb" %}
+{% set version = "1.0.0" %}
 
 package:
-  name: myst-nb
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/m/myst-nb/myst-nb-{{ version }}.tar.gz
-  sha256: 0f61386515fab07c73646adca97fff2f69f41e90d313a260217c5bbe419d858b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
```

Comment:
this is why I don't bother with the `name` junk these days... probably just put it  back to being explict in all places.

Reply:
sounds good

Reply:
```suggestion
  url: https://pypi.io/packages/source/m/myst-nb/myst_nb-{{ version }}.tar.gz
```